docs/content/users/extend/additional-hostnames.md

```diff
@@ -6,13 +6,15 @@ Add additional hostnames to a project in its `.ddev/config.yaml`:
 name: mysite
 
 additional_hostnames:
-- "extraname"
-- "fr.mysite"
-- "es.mysite"
-- "it.mysite"
-- "*.lotsofnames"
+  - "extraname"
+  - "fr.mysite"
+  - "es.mysite"
+  - "it.mysite"
+  - "*.lotsofnames"
 ```
 
+Hostnames can be added with the console command `ddev config --additional-hostnames comma,separated,hostnames`
```

Comment:
I’d move this _below_ the next paragraph so it doesn’t interfere with the flow between the YAML example and the explanation.

```suggestion
```